src/base/utilities/restoreEnvironment.m

```diff
@@ -0,0 +1,53 @@
+function restoreEnvironment(environment)
```

Comment:
May you put here a status, and eventually a `printLevel` for providing information?